rai/show.go

```diff
@@ -127,3 +127,42 @@ func (tx *TransactionResult) Show() {
 		}
 	}
 }
+
+func zip(lists ...[]interface{}) func() []interface{} {
+	zip := make([]interface{}, len(lists))
+	i := 0
+	return func() []interface{} {
+		for j := range lists {
+			if i >= len(lists[j]) {
+				return nil
+			}
+			zip[j] = lists[j][i]
+		}
+		i++
+		return zip
+	}
+}
```

Comment:
🤔 why does this return a function, rather than returning the zip array right away? It seems like you always just call the function once directly, so maybe it could return the array directly now? Maybe this was an artifact of multiple refactorings?

Reply:
ohhh never mind, sorry, i missed that you are calling `iter()` again on each iteration. Never mind. So this is like creating a generator in python. Nifty! :)